src/Couchbase.Lite.Tests.Shared/LogTest.cs

```diff
@@ -268,7 +268,7 @@ public void TestConsoleLoggingLevels()
             WriteLog.To.Database.W("TEST", "TEST WARNING");
             WriteLog.To.Database.E("TEST", "TEST ERROR");
             stringWriter.Flush();
-            stringWriter.ToString().Split(Environment.NewLine, StringSplitOptions.RemoveEmptyEntries).Should()
+            stringWriter.ToString().Split(new[] { Environment.NewLine }, StringSplitOptions.RemoveEmptyEntries).Should()
```

Comment:
Huh....

Reply:
An FYI the API that takes a single string / char came post .NET 4.6.2.  This is the backward compatible version that takes a list of strings or chars.